style.css

```diff
@@ -4,64 +4,116 @@
 @import url("https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;600;700;800&display=swap");
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap');
 
-@media only screen and (max-device-width: 767px) {
-  /*
-  div#wrapper {
-    width: 400px;
+@media only screen and (max-width: 767px) {
```

Comment:
- [OPTIONAL] Your mobile menu works great. :+1: :+1: I'll suggest adding a smooth scroll property in your CSS, it will make navigating to different section transition smoothly. you can add this to the top of your CSS to achieve this. :+1:  `html {
  scroll-behavior: smooth;
}`

- [OPTIONAL] When on a mobile screen, the icon here seems to be broken, I'll suggest you fix that before merging this pull request. :+1: 
- - Screenshot:
![Selection_071](https://github.com/ishimwepati/Portfolio/assets/107632780/bcfc52aa-5130-41d7-825b-68fe80575ae3)

Reply:
Thanks a lot